sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsTest.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Net;
+using System.Threading;
```

Comment:
This one is not needed anymore

Reply:
Thanks for catching this, I will remove :)